deploy/klusterlet/olm-catalog/klusterlet/manifests/klusterlet.clusterserviceversion.yaml

```diff
@@ -12,6 +12,30 @@ metadata:
           },
           "spec": {
             "clusterName": "cluster1",
+            "deployOption": {
+              "mode": "Default"
+            },
+            "externalServerURLs": [
+              {
+                "url": "https://localhost"
+              }
+            ],
+            "namespace": "open-cluster-management-agent",
+            "registrationImagePullSpec": "quay.io/open-cluster-management/registration",
+            "workImagePullSpec": "quay.io/open-cluster-management/work"
+          }
+        },
+        {
+          "apiVersion": "operator.open-cluster-management.io/v1",
+          "kind": "Klusterlet",
+          "metadata": {
+            "name": "klusterlet"
+          },
```

Comment:
do not need detached klusterlet in alm-examples. this example user can apply directly on operatorhub. so we cannot put 2 klusterlet here.